### PR TITLE
Update Views API documentation

### PR DESCRIPTION
**Pull request by JoelMarcey**
Ensure all `props` have documentation. Add more details to current `props`.
Provide more information to the API in general.

> Would like to try to integrate the React Native Web Player for the initial
> example, but not right now.

Test Plan:

http://localhost:8079/react-native/docs/view.html

screencapture-localhost-8079-react-native-docs-view-html-1466648948056

### CODE REVIEW

**Comment by facebook-github-bot-6** [comment]
By analyzing the blame information on this pull request, we identified @janicduplessis and @nicklockwood to be potential reviewers.

**Comment by caabernathy** [comment]
From looking through the image.

Nits although some were in previous doc:

 - wrap the word "Text" in code in the accessibilityLabel prop description.
 - same suggestion as above for the word "true" in the accessible prop description.
 - onMagicTap prop, wrap "true" in code.
 - Typo in onResponderReject description, "another responser"
 - onResponderRelease: the formatting around code in the second sentence of the description appears off.
 - pointerEvents: was in previous docs, but suggest wrapping "View" in code.
 - removeClippedSubviews: perhaps wrap "overflow: hidden" in code.
 - testID: not sure what's preferred, I would say use "Note" instead of "NB"

File: Libraries/Components/View/View.js [hunk 1]
```diff
@@ -8,6 +8,7 @@
  *
  * @providesModule View
  * @flow
+ * @jsdoc
```
Comment (caabernathy) [inline Libraries/Components/View/View.js]: You don't need this as it's a component doc. Only needed for API docs.

**Comment by caabernathy** [comment]
Overall, looks way better with more info. Since my feedback mostly nits, this can be shipped.

**Comment by caabernathy** [comment]
@facebook-github-bot shipit

**Comment by facebook-github-bot-5** [comment]
Thanks for importing. If you are an FB employee go to [Phabricator](https://our.intern.facebook.com/intern/opensource/github/pull_request/291403624538297/int_phab) to review.

**Pull request closed without merging**